internal/setup/setup.go

```diff
@@ -26,13 +26,21 @@ type Release struct {
 	Archives map[string]*Archive
 }
 
+const (
+	ProFIPS        = "fips"
+	ProFIPSUpdates = "fips-updates"
+	ProApps        = "apps"
+	ProInfra       = "infra"
```

Comment:
See the comment about the relationship between an archive and the setup.

Reply:
Done in https://github.com/canonical/chisel/pull/167/commits/e8be8301bed4c96cb4e44b648790c7df091479e8.